iOSMoviesApp/CustomTableViewCells/CastCollectionViewCell/CastCollectionViewCell.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  CastCollectionViewCell.swift
+//  iOSMoviesApp
+//
+//  Created by Yash Agrawal on 11/07/25.
+//
+
+import UIKit
+
+
+protocol CastCollectionViewCellDelegate: AnyObject {
+    func updateProfilePic(with imageData : Data, cacheKey: NSString)
+    func updateProfilePicFromCache(with image: UIImage)
+    func updateProfilePicFromPlaceholder(named: String)
+}
+
+class CastCollectionViewCell: UICollectionViewCell {
+    
+    @IBOutlet weak var realName: UILabel!
+    @IBOutlet weak var fictionalName: UILabel!
+    @IBOutlet weak var castImage: UIImageView!
+    private var castVM : CastViewModel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        // Initialization code
+    }
+    
+    func configureState(_ castData: CastViewModel?) {
+        guard let castData = castData else {return}
```

Comment:
```suggestion
        guard let castData: CastViewModel = castData else {return}
```
Please make it a habit to always specify the type when declaring a variable.

Reply:
Okay, will keep that in mind